modules/ossm-threescale-integrate.adoc

```diff
@@ -11,10 +11,15 @@ You can use these examples to configure requests to your services using the 3sca
 
 .Prerequisites:
 
-* {ProductName} version 1.x
-* A working 3scale account (link:https://www.3scale.net/signup/[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.5/html/installing_3scale/onpremises-installation[3scale 2.5 On-Premises])
+* {ProductName} version 2.x
+* A working 3scale account (link:https://www.3scale.net/signup/[SaaS] or link:https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.9/html/installing_3scale/install-threescale-on-openshift-guide[3scale 2.9 On-Premises])
+* Enabling backend cache requires 3scale 2.9 or greater
 * {ProductName} prerequisites
 * Ensure Mixer policy enforcement is enabled. Update Mixer policy enforcement section provides instructions to check the current Mixer policy enforcement status and enable policy enforcement.
+* Mixer policy and telemetry must be enabled if you are using a mixer plug-in.
+** You will need to properly configure the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/service_mesh/service-mesh-installation#ossm-control-plane-deploy-2x_installing-ossm-v2x[Service Mesh Control Plane (SMCP)] when upgrading.
+
+IMPORTANT: When you want to enable 3scale backend cache with the 3scale Istio adapter, you must also enable mixer policy and mixer telemetry. See xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-control-plane-deploy-1x_installing-ossm-v1x[Deploying the Red Hat OpenShift Service Mesh control plane].
```

Comment:
The link xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-control-plane-deploy-1x_installing-ossm-v1x[Deploying the Red Hat OpenShift Service Mesh control plane] should be xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-control-plane-deploy_installing-ossm. I think this is what's causing the build failure.